libs/forecasting/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": ["plugin:@nrwl/nx/react", "../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
+      "rules": {}
+    },
+    {
+      "files": ["*.ts", "*.tsx"],
+      "rules": {}
```

Comment:
can we add `"max-lines": "error"` in the rules for ts, tsx files
In the refactor work that I was working on, we are trying to limit file lines, so we reduce number of very large files

Reply:
this could be in next PR

Reply:
For new projects we should absolutely do that! It was actually here because it complained at some point when I went past 300 lines 😆 I subsequently broke up my files into smaller ones. Not sure where that rule went, let me check!

Reply:
Ah I think I get it now! This is on with "error" by default via .eslintrc\.eslintrc.custom.eslintrc, and some of the projects have it turned off, but not forecasting, so it is on.